core/src/com/unciv/ui/screens/overviewscreen/EspionageOverviewScreen.kt

```diff
@@ -216,7 +228,37 @@ class EspionageOverviewScreen(val civInfo: Civilization, val worldScreen: WorldS
         selectedSpy = null
         selectedSpyButton?.label?.setText("Move".tr())
         selectedSpyButton = null
-        for ((button, _) in moveSpyHereButtons)
+        for ((button, _) in spyActionButtons)
             button.isVisible = false
     }
+
+    private inner class CoupButton(city: City, isCurrentAction: Boolean) : SpyCityActionButton() {
+        val fist = ImageGetter.getStatIcon("Resistance")
+        init {
+            fist.setSize(24f)
+            add(fist).size(24f)
+            fist.setOrigin(Align.center)
+            if (isCurrentAction) fist.color = Color.WHITE
+            else fist.color = Color.DARK_GRAY
+            onClick {
+                val spy = selectedSpy!!
+                if (!isCurrentAction) {
+                    ConfirmPopup(this@EspionageOverviewScreen, "Do you want to stage a coup in [${city.civ.civName}] with a [${(selectedSpy!!.getCoupChanceOfSuccess(false) * 100f).toInt()}]% chance of success?", "Stage Coup") {
```

Comment:
> Also - we had that situation lately where Kathmandu the Nation only owned Belgrade as City... So better "in [nation]" or "in [city]"?

My intention behind using the Civ name came from coups being nationwide effect since a city-state could own multiple cities. Eg: if it captures another city-state or capital. I can change it though if that seems better.